logit_backward: Port to structured

Freey0 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #61349 OpInfo: bitwise_and
* #60813 bitwise_and: Port to structured
* #60812 bitwise_xor: Port to structured
* #60811 bitwise_or: Port to strucutred
* **#60817 logit_backward: Port to structured**
* #60816 tanh_backward: Port to structured
* #60815 sigmoid_backward: Port to structured
* #60814 xlogy: Port to structured
* #60810 fmin has been ported to the structured kernel, removing the old implementation
* #60809 fmod: Port to structured

Differential Revision: [D29449376](https://our.internmc.facebook.com/intern/diff/D29449376)

Freey0 committed:
logit_backward: Port to structured

[ghstack-poisoned]

Freey0 committed:
Update on "logit_backward: Port to structured"




[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29449376).

Freey0 committed:
Update on "logit_backward: Port to structured"


Differential Revision: [D29449376](https://our.internmc.facebook.com/intern/diff/D29449376)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29449376).

Freey0 committed:
Update on "logit_backward: Port to structured"


Differential Revision: [D29449376](https://our.internmc.facebook.com/intern/diff/D29449376)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29449376).

Freey0 committed:
Update on "logit_backward: Port to structured"


Differential Revision: [D29449376](https://our.internmc.facebook.com/intern/diff/D29449376)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29449376).
